docs/_posts/murat-gunay/2021-12-07-bert_token_classifier_chinese_ner_zh.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: model
+title: Chinese NER Model
+author: John Snow Labs
+name: bert_token_classifier_chinese_ner
+date: 2021-12-07
+tags: [chinese, token_classifier, bert, zh, open_source]
+task: Named Entity Recognition
+language: zh
+edition: Spark NLP 3.3.2
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+- This model is imported from `Hugging Face`.
+
+- It's been fine-tuned for traditional Chinese language, leveraging `bert` embeddings and `BertForTokenClassification`.
+
+## Predicted Entities
+
+`CARDINAL`, `DATE`, `EVENT`, `FAC`, `GPE`, `LANGUAGE`, `LAW`, `LOC`, `MONEY`, `NORP`, `ORDINAL`, `ORG`, `PERCENT`, `PERSON`, `PRODUCT`, `QUANTITY`, `TIME`, `WORK_OF_ART`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/bert_token_classifier_chinese_ner_zh_3.3.2_2.4_1638881767667.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+documentAssembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+sentenceDetector = SentenceDetectorDLModel.pretrained("sentence_detector_dl", "xx")\
+       .setInputCols(["document"])\
+       .setOutputCol("sentence")
+
+tokenizer = Tokenizer()\
+      .setInputCols(["sentence"])\
+      .setOutputCol("token")
+
+tokenClassifier = BertForTokenClassification.pretrained("bert_token_classifier_chinese_ner", "zh"))\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+model = nlpPipeline.fit(empty_data)
+text = """我是莎拉，我从 1999 年 11 月 2 日。开始在斯图加特的梅赛德斯-奔驰公司工作。"""
```

Comment:
Are we sure this sentence is correct? If not, please take one from https://zh.wikipedia.org/